backend/models/symptoms.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const SymptomsSchema = new Schema({
+    symptomName: String
+});
```

Comment:
There is no ID which might be why you currently have an error? Not sure,  but let's look into the bug together and solve it first so we are sure that no breaking code is being pushed to main.

Reply:
no that's urelated. You don't need to add ids to mongo schemas necesarily. Mongo does it on it's own. Also, Aljoscha saw the same issue sometimes.